plugins/axiom/axiom.go

```diff
@@ -0,0 +1,25 @@
+package axiom
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func AxiomCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Axiom CLI",
+		Runs:    []string{"axiom"},
+		DocsURL: sdk.URL("https://axiom.com/docs/cli"),
```

Comment:
This link doesn't look to work, can you please provide the correct one?

I think that should be https://axiom.co/docs/reference/cli